spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/KinesisMessageChannelBinder.java

```diff
@@ -82,8 +81,7 @@ public KinesisMessageChannelBinder(AmazonKinesisAsync amazonKinesis,
 		Assert.notNull(configurationProperties, "'configurationProperties' must not be null");
 		if (ObjectUtils.isEmpty(configurationProperties.getHeaders())) {
 			return BinderHeaders.STANDARD_HEADERS;
-		}
-		else {
+		} else {
```

Comment:
Yeah... Would be great to keep your IDE aligned with our code style if you like to contribute and avoid similar comments 😄 
https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style#braces